python/py-rubicon-objc/Portfile

```diff
@@ -19,13 +19,15 @@ long_description    {*}${description}
 
 homepage            https://beeware.org/project/projects/bridges/rubicon
 
-checksums           rmd160  8f1ac44ccf1cb06e8c3f2cfa417bc608e7992f02 \
-                    sha256  be937d864bd1229f860defabb89b40c53634eedc36448d89ad3c14eb3286e509 \
-                    size    168431
+checksums           rmd160  e2922af14dfb778af34b45741dc2d0713337c654 \
+                    sha256  f7bede0f0e73dde07083785984c75b3acf76116298743758ea5c51b3ebb81978 \
+                    size    170454
 
 if {${name} ne ${subport}} {
+    patchfiles-append   patch-pyproject.diff
+
     depends_lib-append  port:pre-commit \
                         port:py${python.version}-pytest \
                         port:py${python.version}-setuptools_scm \
-                        port:py${python.version}-tox \
+                        port:py${python.version}-tox
```

Comment:
very unlikely that `py-tox` is a library dependency. Typically, it's only a test dependency and I don't see any evidence that that would be different here.

Reply:
Looks like that is the case indeed. I dropped unneeded deps now.